caikit/runtime/interceptors/caikit_runtime_server_wrapper.py

```diff
@@ -379,3 +384,63 @@ def _get_handler_fn(handler: RpcMethodHandler) -> Callable:
         if handler.stream_unary:
             return handler.stream_unary
         return handler.stream_stream
+
+
+class CaikitRuntimeServerMultiFuncWrapper(CaikitRuntimeServerWrapper):
+    """This class has the same functionality of the CaikitRuntimeServerWrapper
```

Comment:
Why not extend the existing `CaikitRuntimeServerWrapper` to allow this?

Reply:
Uhhh good question, I didn't want to break anything 😅. It would probably be a pretty straightforward change so let me refactor